GitUI/UserControls/GravatarControl.cs

```diff
@@ -85,13 +92,17 @@ private void RegisterAtGravatarcomToolStripMenuItemClick(object sender, EventArg
             }
         }
 
-        private async void ClearImagecacheToolStripMenuItemClick(object sender, EventArgs e)
+        private void ClearImagecacheToolStripMenuItemClick(object sender, EventArgs e)
```

Comment:
This change will change behavior from crashing on exceptions to swallowing them. 
In VS we have a handle "FileAndForget" extension method on JoinableTask and Task so that we can just tack a call to it on the end of JTF.RunAsync or Task.Run calls and it handles failures (either by filing telemetry so the failure will get noticed by authors, or displaying an error, etc.). You might consider that.

Reply:
@sharwell Perhpas you overlooked that comment.